idaes/models/properties/general_helmholtz/components/parameters/h2o.json

```diff
@@ -0,0 +1,368 @@
+{
```

Comment:
Would it make more sense (and make it easier to find things) if each component was put into a sub-folder with the component name?

I could see this folder getting very large as new components are added.

Reply:
Yeah, I considered it, but it would require modifying the external functions.  Right now there is now setting for parameter file path.  I should probably set it per component or set it up to look for subdirectories.  I want to make some other changes like reference state options, but I'm not up to changing the external functions and building new binaries right now.